projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -4,19 +4,19 @@
 
 ## 13.3-a.7 - 2024-03-25
 ### Enhancements
-- Block category "Earn" renamed to "Monetize" [#36480]
-- Jetpack AI: when response includes a title and post title is empty, use provided title as post title [#36500]
-- Removed Like button from master bar [#36456]
-- Reorder newsletter settings cards to improve hierarchy [#36465]
-- Trigger red bubble notifiction on My Jetpack when bad install is detected [#36449]
-- Use radio controls instead of toggles on Email Settings [#36532]
+- Blocks: "Earn" category renamed to "Monetize". [#36480]
+- Jetpack AI: When the response includes a title and post title is empty, use provided title as post title. [#36500]
+- My Jetpack: Trigger red bubble notifiction when a broken installation is detected. [#36449]
+- Newsletters: Reorder settings cards to improve hierarchy. [#36465]
+- Newsletters: Use radio controls instead of toggles on Email Settings. [#36532]
+- Sharing: Removed Like button from master bar. [#36456]
```

Comment:
```suggestion
- Blocks: "Earn" category renamed to "Monetize". [#36480]
- Jetpack AI: When the response includes a title and post title is empty, use provided title as post title. [#36500]
- My Jetpack: Trigger red bubble notification when a broken installation is detected. [#36449]
- Newsletters: Reorder settings cards to improve hierarchy. [#36465]
- Newsletters: Use radio buttons instead of toggles on Email Settings. [#36532]
- Sharing: Remove Like button from master bar. [#36456]
```